### DISCUSSION TITLE
Delete/remove network from UI?

### DISCUSSION THREAD

**Discussion by genpfault** [pacien/tincapp#125] [Q&A] [answered]
Is there any way to delete/remove a network via the UI?  Short of clearing the app data, of course :)

I tried long-pressing a network name but that didn't do anything.  I also wasn't seeing anything obvious in the 'wrench' menu.

**Comment by pacien** [comment, marked as answer]
Hi!

Sadly, there is currently no way to remove a network configuration
folder from the GUI. Any edition or deletion of the config has to be
done externally, at least for now (until the UI gets enriched for that,
issue #30).
